lmnet/lmnet/networks/base.py

```diff
@@ -58,7 +58,7 @@ def __init__(
         self.optimizer_kwargs = optimizer_kwargs if optimizer_kwargs is not None else {'learning_rate': 0.01}
         self.learning_rate_func = learning_rate_func
         self.learning_rate_kwargs = learning_rate_kwargs if learning_rate_kwargs is not None else {}
-        self.classes = classes
+        self.classes = list(map(lambda _class: _class.replace(' ', '_'), classes))
```

Comment:
@Joeper214 
Why convert with `list` is needed?

Reply:
Oh. I was thinking of parsing the `map` object back to `list`. but it seems not necessary thanks for pointing out.

Reply:
Updated in next commit.

Reply:
I checked the test case it seems that test fails if we don't parse the `map` object back to `list`.
```

[2019-10-08T01:54:22Z]         self.classes = classes
--
  | [2019-10-08T01:54:22Z] >       self.num_classes = len(classes)
  | [2019-10-08T01:54:22Z] E       TypeError: object of type 'map' has no len()
  | [2019-10-08T01:54:22Z]
  | [2019-10-08T01:54:22Z] lmnet/networks/object_detection/yolo_v2.py:1044: TypeError
```

Reply:
I see.